core/.storage-layout.md

```diff
@@ -26,6 +26,8 @@ forge inspect --pretty src/Factory.sol:Factory storage-layout
 |---------------|---------------------------------------------------------------|------|--------|-------|-------------------------|
 | getMarket     | mapping(contract IUniswapV3Pool => struct Factory.Market)     | 0    | 0      | 32    | src/Factory.sol:Factory |
 | getParameters | mapping(contract IUniswapV3Pool => struct Factory.Parameters) | 1    | 0      | 32    | src/Factory.sol:Factory |
-| isBorrower    | mapping(address => bool)                                      | 2    | 0      | 32    | src/Factory.sol:Factory |
-| couriers      | mapping(uint32 => struct Factory.Courier)                     | 3    | 0      | 32    | src/Factory.sol:Factory |
+| isLender      | mapping(address => bool)                                      | 2    | 0      | 32    | src/Factory.sol:Factory |
+| isBorrower    | mapping(address => bool)                                      | 3    | 0      | 32    | src/Factory.sol:Factory |
+| couriers      | mapping(uint32 => struct Factory.Courier)                     | 4    | 0      | 32    | src/Factory.sol:Factory |
+| isCourier     | mapping(address => bool)                                      | 5    | 0      | 32    | src/Factory.sol:Factory |
```

Comment:
Unrelated. I just forgot to run the layout script a while back, and this file has been out of date ever since.